src/content/app/tools/blast/views/blast-submissions/BlastSubmissions.scss

```diff
@@ -1,7 +1,15 @@
 @import 'src/styles/common';
 
+.submissionsContainer {
+  display: grid;
+  grid-template-rows: auto auto minmax(0, 1fr);
+  height: 100%;
+}
+
 .container {
   padding: 0 30px $global-padding-bottom 60px; // TODO: these 30px, 60px and 90px should either become a reusable constant or be included in some sort of BlastMainContainer component
```

Comment:
Is this TODO still relevant for us?

Reply:
I think so, we haven't done a design review for blast yet and AFAIK once we have the padding finalised then we can make them constant. @azangru do you remember anything about it?

Reply:
I remember writing the TODO 😄  30px and 60px look very suspicious now — are they our right padding and left padding that Jyo has added as variables?